src/main/kotlin/com/github/pool_party/pull_party_bot/commands/PingCommandHandlers.kt

```diff
@@ -38,7 +38,7 @@ fun Bot.handleStart(msg: Message) {
  * Return the help message.
  */
 fun Bot.handleHelp(msg: Message, args: String?) {
-    val parsedArgs = parseArgs(args)
+    val parsedArgs = parseArgs(args)?.distinct()
```

Comment:
Shouldn't it be added in `parseArgs`?

Reply:
party pulling needs all arguments with repetition to handle `/create user-name user-name` case

Reply:
🤪